### DISCUSSION TITLE
Investigate adaptive SMC samplers

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1543] [New methods]
See [here](https://projecteuclid.org/euclid.ba/1369407558).

**Comment by MichaelClerx** [comment]
~Can we close #120 then in favour of this ticket?~ No

**Comment by ben18785** [comment]
Sure.

On Mon, Mar 26, 2018 at 1:26 PM, Michael Clerx <notifications@github.com>
wrote:

> Can we close #120 <https://github.com/pints-team/pints/issues/120> then
> in favour of this ticket?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pints-team/pints/issues/279#issuecomment-376148037>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AESFqOg-CQnjwAyU30oC1_aEqhE66qbNks5tiN6AgaJpZM4S5XuX>
> .
>



-- 
Best,

Ben

**Comment by MichaelClerx** [comment]
Just for reference: https://github.com/pints-team/pints/tree/issue%23120-sequential-Monte-Carlo
